lib/contentful_middleman/helpers.rb

```diff
@@ -7,7 +7,8 @@ def contentful_instances
     end
 
     def localize_entry(entry, locale, fallback_locale='en-US')
-      localized_entry = {}
+      localized_entry = entry.class.new
+      localized_entry = {} unless localized_entry.is_a? ::Hash
```

Comment:
Great, wasn't sure on that.

Did you want Thor in the `gemfile`, or happy with it being a very integral part of `middleman-core`?  If Middleman dropped Thor, plenty of this might break anyways..